### ISSUE TITLE
Cannot install without git

### ISSUE THREAD

**Issue by yamikuronue** [krampstudio/grunt-jsdoc#126]
Despite the changelog saying that the dependencies had moved to npm packages instead of git, I'm unable to install this package without git on my build server. We use svn at my place of business, and do not typically have git installed on build servers.

We're including this via "grunt-jsdoc": "~0.5.7", and it doesn't look like 0.5.8 has any change to this issue from the changelog. npm install fails with:

```
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/dominictarr/crypto-browserify.git undefined
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/dominictarr/crypto-browserify.git undefined
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/hegemonic/taffydb.git undefined
error	23-Mar-2015 08:53:59	npm ERR! git clone https://github.com/hegemonic/taffydb.git undefined
error	23-Mar-2015 08:54:08	npm ERR! Linux 2.6.32-220.23.1.el6.x86_64
error	23-Mar-2015 08:54:08	npm ERR! argv "node" "/usr/bin/npm" "install"
error	23-Mar-2015 08:54:08	npm ERR! node v0.10.33
error	23-Mar-2015 08:54:08	npm ERR! npm  v2.7.0
error	23-Mar-2015 08:54:08	npm ERR! code ENOGIT
error	23-Mar-2015 08:54:08	
error	23-Mar-2015 08:54:08	npm ERR! not found: git
error	23-Mar-2015 08:54:08	npm ERR! 
error	23-Mar-2015 08:54:08	npm ERR! Failed using git.
error	23-Mar-2015 08:54:08	npm ERR! This is most likely not a problem with npm itself.
error	23-Mar-2015 08:54:08	npm ERR! Please check if you have git installed and in your PATH.
```

**Comment by yamikuronue** [comment]
Nevermind, it's not this repo that's causing it, but jsdoc itself

**Comment by yamikuronue** [comment]
Reopening because the version number is pegged at 3.2.X, while the version of jsdoc that fixes this issue for me is 3.3. If you could change your ~ to a >, I could explicitly require jsdoc >3.3 and therefore fix this issue.

**Comment by krampstudio** [comment]
Hi, this is intentional. 

You can use the plugin in beta version if you want jsdoc 3.3.x. :

`npm install grunt-jsdoc@beta`

3.2.x is their current stable.
